lib/credit_card_sanitizer.rb

```diff
@@ -5,33 +5,45 @@
 class CreditCardSanitizer
   # https://github.com/Shopify/active_merchant/blob/master/lib/active_merchant/billing/credit_card_methods.rb#L5-L18
```

Comment:
```suggestion
  # https://github.com/activemerchant/active_merchant/blob/e282efb447bf20688254301fda0534d6201f97f1/lib/active_merchant/billing/credit_card_methods.rb#L7-L58
```
but we seem to follow another list now.

Reply:
I have updated this URL as you suggested. Let me know if you want any missing entries from this new list added.

Reply:
Well, we should probably put the URL source for the actual code we use. I have seen the same regular expressions almost identically a few places around the internet, but the only source I see is “requested by customer” 😅

Reply:
They were requested by a customer: https://zendesk.slack.com/archives/C05L0J3D690/p1714122051881239
Our original regexes haven't been altered, only updated to include any edge cases that the customer thought was missing with that provided list.